lib/facter/nomad_version.rb

```diff
@@ -1,5 +1,6 @@
 # nomad_version.rb
-
+#
+# @param [String] nomad_version
```

Comment:
Does this work for facts? The fact doesn't have a parameter, right?

Reply:
it seems it doenst, i will remove